ios/FluentUI/People Picker/InitialsView.swift

```diff
@@ -39,11 +39,6 @@ class InitialsView: UIView {
             initials = String(secondaryText.prefix(1))
         }
 
-        // Fallback to `#` otherwise
```

Comment:
Do we really want to get rid of this? Can InitialsView be used without AvatarView? If so, we might want to keep this logic, or not, depends.

Reply:
while this view could be used somewhere else, (ignoring the fact that background color is using Colors.avatarBackgroundColors) does it make sense that the character # used for other labels and stuff?

Reply:
I'm not sure why "#" was added in the first place so I don't have a good answer for that.

Reply:
i think this was there becuase... outlook does it that way? i can't think of good reason to keep it. lets get rid of it unless you think it will be useful for the future?

Reply:
I agree, let's get rid of it.